src/REPLMode/REPLMode.jl

```diff
@@ -374,7 +374,7 @@ end
 #############
 function do_cmd(repl::REPL.AbstractREPL, input::String; do_rethrow=false)
     if !isinteractive() && !TEST_MODE[] && !PRINTED_REPL_WARNING[]
-        @warn "The Pkg REPL interface is intended for interactive use, use with caution from scripts."
+        @warn "The Pkg REPL mode is intended for interactive use, use with caution from scripts, or use the functional API."
```

Comment:
above you use "interface" but change it here

Reply:
Do you prefer mode or interface?

Reply:
mode I guess

Reply:
The only place I see where `REPL interface` remains is in the table where it was `REPL version` previously. Is that what you mean? I'll change it to `REPL mode` in the table if so.